index.d.ts type-error

```
node_modules/bee-queue/index.d.ts:13:40 - error TS2314: Generic type 'Job<T>' requires 1 type argument(s).

13   backoffStrategies: Map<string, (job: BeeQueue.Job) => number>;
                                          ~~~~~~~~~~~~

node_modules/bee-queue/index.d.ts:120:30 - error TS2304: Cannot find name 'this'.

120       strategy: keyof typeof this.backoffStrategies,
                                 ~~~~

Found 2 errors.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -10,7 +10,7 @@ declare class BeeQueue<T = any> extends EventEmitter {
   jobs: any;
   paused: boolean;
   settings: any;
-  backoffStrategies: Map<string, (job: BeeQueue.Job) => number>;
+  backoffStrategies: Map<string, (job: BeeQueue.Job<T>) => number>;
 
   constructor(name: string, settings?: BeeQueue.QueueSettings);
 
@@ -116,10 +116,7 @@ declare namespace BeeQueue {
 
     setId(id: string): this;
     retries(n: number): this;
-    backoff(
-      strategy: keyof typeof this.backoffStrategies,
-      delayFactor?: number
-    ): this;
+    backoff(strategy: string, delayFactor?: number): this;
     delayUntil(dateOrTimestamp: Date | number): this;
     timeout(milliseconds: number): this;
     save(): Promise<this>;

